### ISSUE TITLE
Add an option to not return after retreating

### ISSUE THREAD

**Issue by GoogleFrog** [ZeroK-RTS/Zero-K#3581]
Some people would like a way for units to not leave their retreat zone upon entering it. Perhaps it could be a toggle somewhere. Perhaps retreat zones could have command queues similar to airpads.

**Comment by Sgathec** [comment]
Yes, I would like to prevent returning as well. Is the player able to easily stop a single unit from retreating too?
